src/component/LoginForm.vue

```diff
@@ -58,7 +59,7 @@ export default {
     /*height: 200px;*/
     /*max-height: 30%;*/
 
-    background-color: rgba(255, 255, 255, 0.5);
+    background-color:transparentize( $shadow, 0.5);
```

Comment:
Same as above, why don't we create a new variable to save this `transparentize($shadow, 0.5)` as a new color variable?